batcher/aligned-batcher/src/lib.rs

```diff
@@ -1786,7 +1795,12 @@ impl Batcher {
             ETHEREUM_CALL_MAX_RETRY_DELAY,
         )
         .await
-        .map_err(|e| BatcherError::BatchUploadError(e.to_string()))
+        .map_err(|e| BatcherError::BatchUploadError(e.to_string()))?;
+
+        self.metrics
+            .s3_latency
+            .set(start.elapsed().as_micros() as i64);
```

Comment:
Here you are using `start.elapsed().as_micros()` instead of `start.elapsed().as_millis()` like in the other two gauges. Anyways, it seems to work fine. Is there a reason for that? Is it because the S3 response latency is some orders of magnitude lower than that of the other two methods?

Reply:
Exactly, It was only for better precision.